cabal-testsuite/PackageTests/Outdated/Issue8283/cabal.out

```diff
@@ -0,0 +1,45 @@
+# cabal v2-update
+Downloading the latest package list from test-local-repo
+# cabal outdated
+Configuration is affected by the following files:
+- cabal.project
+Outdated dependencies:
+* package-a
+  - lib:package-a
+    * template-haskell >=2.3 && <2.4 (latest: 2.4.0.0)
+    * template-haskell >=2.3.0 && <2.3.1 (latest: 2.4.0.0)
+* package-b
+  - lib:package-b
+    * base >=3.0 && <3.5 (latest: 4.0.0.0)
+# cabal v2-outdated
+Configuration is affected by the following files:
+- cabal.project
+Outdated dependencies:
+* package-a
+  - lib:package-a
+    * template-haskell >=2.3 && <2.4 (latest: 2.4.0.0)
+    * template-haskell >=2.3.0 && <2.3.1 (latest: 2.4.0.0)
+# cabal v2-outdated
+Configuration is affected by the following files:
+- cabal.project
+template-haskell
+template-haskell
+base
```

Comment:
Would it make sense to nub this `--simple-output` list (and order it alphabetically)?